client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -116,6 +118,23 @@ const Settings = ( {
 		);
 	};
 
+	const renderStatusSection = () => {
+		if ( ! ( domain && isDomainOnly ) ) {
+			return null;
+		}
+
+		return (
+			<Accordion
+				title={ translate( 'Domain status', { textOnly: true } ) }
+				subtitle={ translate( 'Your domain is parked', { textOnly: true } ) }
```

Comment:
I know this is what is on the design file, but since we also have the language in the paragraph below explaining that this domain ins't connected to a WPCOM site, I think we should actually just use the same status as we show in the domain list here "Active", etc.